sbc_compassion/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_sponsorship_correspondence","Read only access on sponsorship_correspondence","model_sponsorship_correspondence","child_compassion.group_sponsorship",1,0,0,0
+"access_res_lang_compassion","Read only access on res_lang_compassion","model_res_lang_compassion","child_compassion.group_sponsorship",1,0,0,0
+"access_sponsorship_correspondence","Full access on sponsorship_correspondence","model_sponsorship_correspondence","child_compassion.group_admin",1,1,1,1
```

Comment:
You cannot have twice the same rule id ("access_sponsorship_correspondence" and "access_res_lang_compassion"). Rename them. Moreover, `model_sponsosrship_correspondence` can have full access for group_sponsorship. I was only talking about `res_lang_compassion` when I told to separate admin rights.